src/parsers/js/transformers/eslint2/index.js

```diff
@@ -0,0 +1,23 @@
+import {defineRule, runRule} from '../../utils/eslintUtils';
+import pkg from 'eslint2/node_modules/eslint/package.json';
+
+const ID = 'eslint-v2';
+const name = 'ESLint v2'
+
+export default {
+  id: ID,
+  displayName: name,
+  version: pkg.version,
+  homepage: pkg.homepage,
+
+  defaultParserID: 'acorn-to-esprima',
+
+  loadTransformer(callback) {
+    require(['eslint2'], callback);
+  },
+
+  transform({ eslint, rules, sourceCode }, transformCode, code) {
+    defineRule(rules, transformCode);
+    return runRule(code, eslint, sourceCode);
```

Comment:
I had trouble getting espree to run properly (it kept giving me syntax errors for some reason) so I just gave in and used babel-eslint like the ESLint 1 transformer does. I guess it doesn't really matter which parser is used as ESLint can just use the AST.